### PR TITLE
91 create sprint component

### PR DESCRIPTION
**Pull request by Vadis00**

### CODE REVIEW

File: frontend/src/app/components/backlog/sprint/sprint.component.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import { UserModel } from 'src/core/models/user/user-model';
+import { UserCircle } from 'src/shared/components/tasque-team-select/models';
+
+@Component({
+  selector: 'app-sprint',
+  templateUrl: './sprint.component.html',
+  styleUrls: ['./sprint.component.sass'],
+})
+export class SprintComponent implements OnInit, OnChanges {
+  @Input() public sprint: SprintModel;
```
Comment (ismv) [inline frontend/src/app/components/backlog/sprint/sprint.component.ts:25]: please describe input/output's purpose externally via comment

File: frontend/src/app/components/backlog/sprint/issue/issue.component.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { UserModel } from 'src/core/models/user/user-model';
+import { SprintService } from 'src/core/services/sprint.service';
+
+@Component({
+  selector: 'app-issue',
+  templateUrl: './issue.component.html',
+  styleUrls: ['./issue.component.sass'],
+})
+export class IssueComponent implements OnInit {
+  @Input() public issue: TaskModel;
```
Comment (ismv) [inline frontend/src/app/components/backlog/sprint/issue/issue.component.ts:17]: please describe input/output's purpose externally via comment

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 3]
```diff
@@ -11,44 +11,17 @@ <h4 class="task-creation-header-title">Create issue</h4>
...
           <div class="input-wrapper">
-            <tasque-dropdown
-              label="Project"
-              [options]="projects"
-              formControlName="projectControl"
-              [errorMessage]="projectErrorMessage"
-              [invalid]="!projectControl.valid"
-            >
-            </tasque-dropdown>
+            <tasque-dropdown label="Project" [options]="projects" formControlName="projectControl" [errorMessage]="projectErrorMessage" [invalid]="!projectControl.valid"> </tasque-dropdown>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: formatting

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 4]
```diff
@@ -11,44 +11,17 @@ <h4 class="task-creation-header-title">Create issue</h4>
...
           <div class="input-wrapper">
-            <tasque-dropdown
-              label="Issue type"
-              [options]="issueTypes"
-              [formControl]="issueTypeControl"
-              [errorMessage]="issueTypeErrorMessage"
-              [invalid]="!issueTypeControl.valid"
-            >
-            </tasque-dropdown>
+            <tasque-dropdown label="Issue type" [options]="issueTypes" [formControl]="issueTypeControl" [errorMessage]="issueTypeErrorMessage" [invalid]="!issueTypeControl.valid"> </tasque-dropdown>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: formatting

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 5]
```diff
@@ -11,44 +11,17 @@ <h4 class="task-creation-header-title">Create issue</h4>
...
           <div class="input-wrapper">
-            <tasque-input
-              label="Summary"
-              [formControl]="summaryControl"
-              [errorMessage]="summaryErrorMessage"
-              [invalid]="!summaryControl.valid"
-              ngDefaultControl
-            >
-            </tasque-input>
+            <tasque-input label="Summary" [formControl]="summaryControl" [errorMessage]="summaryErrorMessage" [invalid]="!summaryControl.valid" ngDefaultControl> </tasque-input>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: formatting

File: frontend/src/shared/components/tasque-task-creation/task-creation.component.html [hunk 6]
```diff
@@ -11,44 +11,17 @@ <h4 class="task-creation-header-title">Create issue</h4>
...
           <div class="editor-wrapper">
             <label class="label">Description</label>
-            <angular-editor
-              class="editor-conteiner"
-              [(ngModel)]="editorContent"
-              [config]="editorConfig"
-              formControlName="descriptionControl"
-            >
-            </angular-editor>
+            <angular-editor class="editor-conteiner" [(ngModel)]="editorContent" [config]="editorConfig" formControlName="descriptionControl"> </angular-editor>
```
Comment (ismv) [inline frontend/src/shared/components/tasque-task-creation/task-creation.component.html]: formatting

**Review by ismv — COMMENTED** [review]
Conflicts Resolving

**Review by ismv — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**